experiments/modern/src/App.js

```diff
@@ -49,11 +59,18 @@ async function getSkin() {
     // TODO: This is probalby only valid if in an `<elements>` node
     switch (node.name) {
       case "bitmap": {
-        const { file, gammagroup, h, id, w, x, y } = node.attributes;
+        let { file, gammagroup, h, id, w, x, y } = node.attributes;
         // TODO: Escape file for regex
         const img = Utils.getCaseInsensitveFile(zip, file);
         const imgBlob = await img.async("blob");
         const imgUrl = URL.createObjectURL(imgBlob);
+        if (w  === undefined || h  === undefined) {
+          let [width, height] = await loadImage(imgUrl);
+          w = width;
+          h = height;
+          x = 0;
+          y = 0;
```

Comment:
Is this (zeroing x,y) correct? I wonder if it’s possible to have an image with no height/width, that still specified the position.

Reply:
yea, I was assuming that if we didn't have w/h that we wouldn't have x/y, so zeroing...but now that you ask, probably should just check